config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  devise_for :users
+  resources :spendings, only: %i[index new]
+  resources :types, only: %i[index new]
+  resources :users, only: %i[index]
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
```

Comment:
- [ ] It looks like after lately introduced changes the application got some bugs. When I try to create a new type, I'm getting no route error: 
<img width="339" alt="Screenshot 2022-05-27 at 17 03 17" src="https://user-images.githubusercontent.com/77038610/170704579-5af99380-3379-47e9-ba90-fe85fd070746.png">

Kindly check fix the issue and test other functionalities.